app/controllers/mixins/containers_common_mixin.rb

```diff
@@ -36,11 +36,83 @@ def show_container(record, controller_name, display_name)
+      @view, @pages = get_view(ContainerReplicator, :parent => record) # Get the records (into a view) and the paginator
+      @showtype = @display
+      if @view.extras.fetch(:total_count, 0) > @view.extras.fetch(:auth_count, 0)
+        @bottom_msg = "* You are not authorized to view "
+        + pluralize(@view.extras[:total_count] - @view.extras[:auth_count], "other #{title.singularize}")
+        + " on this " + ui_lookup(:tables => @table_name)
+      end
+    elsif @display == "container_projects" || session[:display] == "container_projects" && params[:display].nil?
+      title = ui_lookup(:tables => "container_projects")
+      drop_breadcrumb(:name => record.name + " (All #{title})",
+                      :url  => "/#{controller_name}/show/#{record.id}?display=#{@display}")
+      @view, @pages = get_view(ContainerProject, :parent => record)  # Get the records (into a view) and the paginator
+      @showtype = @display
+      if @view.extras.fetch(:total_count, 0) > @view.extras.fetch(:auth_count, 0)
+        @bottom_msg = "* You are not authorized to view "
+        + pluralize(@view.extras[:total_count] - @view.extras[:auth_count], "other #{title.singularize}")
+        + " on this " + ui_lookup(:tables => @table_name)
+      end
     elsif @display == "containers"
-      show_container_display("containers", Container, "container_group")
+      title = ui_lookup(:tables => "containers")
+      drop_breadcrumb(:name => record.name + " (All #{title})",
+                      :url  => "/container_group/show/#{record.id}?display=#{@display}")
+      @view, @pages = get_view(Container, :parent => record) # Get the records (into a view) and the paginator
+      @showtype = @display
+      if @view.extras.fetch(:total_count, 0) > @view.extras.fetch(:auth_count, 0)
+        @bottom_msg = "* You are not authorized to view " +
+        pluralize(@view.extras[:total_count] - @view.extras[:auth_count], "other #{title.singularize}")
+        + " on this " + ui_lookup(:tables => @table_name)
+      end
     elsif @display == "container_services" || session[:display] == "container_services" && params[:display].nil?
-      show_container_display("container_services", ContainerService)
+      title = ui_lookup(:tables => "container_services")
+      drop_breadcrumb(:name => record.name + " (All #{title})",
+                      :url  => "/#{controller_name}/show/#{record.id}?display=#{@display}")
+      @view, @pages = get_view(ContainerService, :parent => record)  # Get the records (into a view) and the paginator
+      @showtype = @display
+      if @view.extras[:total_count] > @view.extras[:auth_count] && @view.extras[:total_count] &&
+         @view.extras[:auth_count]
+        count_text = pluralize(@view.extras[:total_count] - @view.extras[:auth_count], "other #{title.singularize}")
+        @bottom_msg = "* You are not authorized to view #{count_text} on this #{ui_lookup(:tables => @table_name)}"
+      end
```

Comment:
@moolitayer this is not the rebase we were looking for. You shouldn't have reverted @kbrock work. You should have used his new method.